consumer/start-consumer.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# set image and container names
+IMAGE_NAME="kafka-consumer"
+CONTAINER_NAME="kafka-consumer-container"
+
+# Check if the Docker image already exists
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" == "" ]]; then
+    echo "Building the Docker image..."
+    docker build -t $IMAGE_NAME .
+else
+    echo "Docker image already exists. Skipping build."
+}
+
+# Check if the container is already running
+if [[ "$(docker ps -q -f name=$CONTAINER_NAME)" ]]; then
```

Comment:
this option should be managed by a flag to force the restart

Reply:
done